server/roomlogs.ts

```diff
@@ -156,15 +156,23 @@ export class Roomlog {
 		if (!Roomlogs.rollLogTimer) void Roomlogs.rollLogs();
 	}
 	add(message: string) {
-		if (message.startsWith('|uhtmlchange|')) return this.uhtmlchange(message);
-		this.roomlog(message);
-		if (this.logTimes && message.startsWith('|c|')) {
-			message = '|c:|' + (~~(Date.now() / 1000)) + '|' + message.substr(3);
+		if (message.startsWith('|uhtmlchange|')) {
```

Comment:
I would probably remove this magic, if one uses a function call and the other doesn't. I'd just update them so they all both function calls.

Reply:
There are plenty of `this.room.add` calls. I want to limit the changes to minimum, considering current implementation causes issues when connecting to Showdown (you have to refresh the page multiple times sometimes).

Reply:
This is really just a bandwidth optimization; I don't think third party server code being slightly suboptimal is worth having this unintuitive API in place. If you _really_ care about diff size, I can do this change in a separate commit, but it seems like the wrong thing to optimize for.

Reply:
Fixed.